R/tbl_ae.R

```diff
@@ -33,6 +33,9 @@
 #' @param zero_symbol String used to represent cells with zero counts. Default
 #' is the em-dash (`"\U2014"`). Using `zero_symbol = NULL` will print the
 #' zero count statistics, e.g. `"0 (0)"`
+#' @param sort Controls order AEs and SOCs appear in the tables. Default is
+#'  `"alphanumeric"`. `sort = "frequency"` will sort in decreasing frequency
+#'  order.
```

Comment:
The `sort` argument applies only to ae overall (when ae is specified) or ae within soc (when both ae and soc are specified).  Maybe the documentation could be amended like:

```suggestion
#' @param sort Controls order AEs appear.  When `ae` is specified without `soc`, applies to AEs overall. When `ae` is #' specified with `soc`, applies to AEs within SOCs.  Default is
#'  `"alphanumeric"` for alphabetical sorting. `sort = "frequency"` will sort in decreasing frequency
#'  order.
```

Reply:
Or I am wondering if I tested this enough to understand what was happening...